mongo/src/main/java/com/redhat/lightblue/mongo/crud/MongoCRUDController.java

```diff
@@ -1108,4 +1107,29 @@ private Error analyzeException(Exception e, final String otherwise, final String
             return Error.get(otherwise, msg);
         }
     }
-}
+
+    @Override
+    public CRUDHealth checkHealth() {
+        boolean isHealthy = true;
+        Collection<MongoConfiguration> configs = dbResolver.getConfigurations();
+        List<String> details = new ArrayList<>(configs.size());
+        DBObject ping = new BasicDBObject("ping", 1);
+
+        for (MongoConfiguration config : configs) {
+            try {
+                CommandResult result = config.getDB().command(ping);
+                if (!result.get("ok").equals(1.0)) {
+                    isHealthy = false;
+                    details.add(new StringBuilder(config.toString()).append("\n").append("ping:NOK").toString());
```

Comment:
Agree with Sunny. Also, newlines aren't rendered in JSON strings, so it might be better to use a different separator than a newline for readability, maybe : or = or ->, something like that.